make.sh

```diff
@@ -9,15 +9,17 @@ rm -rf *.eep
 ## Enabled DEVICES
 ##
 ##   DS18B20  (1w    | temperature)
-##   SHT21    (i2c   | temperature + humidity)
-##   SHT31    (i2c   | temperature + humidity + calbibration)
+##   SHT21    (i2c   | temperature + humidity + optional calbibration)
```

Comment:
small typo: "calibration"

Reply:
i removed the .hex and .eep files and did fixed the typos 👍